### PR TITLE
Article: Volunteer Project Management

### PR DESCRIPTION
**Pull request by jasonsreynolds**
Creates a volunteer project management wiki page.

Adds an initial volunteer project management wiki page that outlines the current process and tools being used.

### CODE REVIEW

File: Volunteer-Project-Management.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
++ **Project Scope** - This list provides a general outline of what the project should accomplish.
++ **User Stories** - These are the <em>specific</em> features that the development must include.
++ **In Progress** - These are the user stories that the team is currently working on.
++ **Recently Completed** - These are the user stories that the team has completed since the last project conference.
++ **Demo'd** - These are the user stories that have been demo'd to, and accepted by, the client.
+
+___
+
+### Organizing a Scope Call
+In Progress.
```
Comment (Rafase282) [inline Volunteer-Project-Management.md]: Are you still working on this?

Comment (jasonsreynolds) [inline Volunteer-Project-Management.md]: Yup. Waiting for feedback from my fellow volunteer PMs.

Comment (SeanFCC) [inline Volunteer-Project-Management.md]: Jason, what feedback are you looking for?

@Rafase282 ,
If there is no issue with the current PR can we commit and then we can work together ( Jason, the other PMs, and myself) to add additional information?

**Comment by Rafase282** [comment]
Merged, just add the rest when you get the feedback, this way others can chime in too.

**Comment by abhisekp** [comment]
@jasonsreynolds is this related to https://github.com/FreeCodeCamp/wiki/issues/667 ? If so, then which task?

**Comment by jasonsreynolds** [comment]
@Rafase282 Thanks.
@abhisekp It isn't directly related. Those articles are more Project Management in general (to educate the campers and stakeholders), while this is the specific methodology we're using to manage the Free Code Camp volunteer projects (for use as a reference by the volunteer PMs).

**Comment by abhisekp** [comment]
@jasonsreynolds cool :sunglasses: 
Thanks for the article. :tada: :smiley:

**Pull request merged**